src/constants.ts

```diff
@@ -1,6 +1,33 @@
+import {
+  OBJECTIONS_OBJECTOR_ORGANISATION,
+  STRIKE_OFF_OBJECTIONS } from "./model/page.urls";
+
 export const APP_NAME = "strike-off-objections-web";
 export const OBJECTIONS_SESSION_NAME = "strike_off_objections_session";
 export const SESSION_COMPANY_PROFILE = "objections_company_profile";
 export const SESSION_OBJECTION_CREATE = "objection_create";
 export const SESSION_OBJECTION_ID = "objection_id";
+export const SESSION_OBJECTOR = "objector";
 export const CHANGE_ANSWER_KEY = "change";
+
+
+export const FULL_NAME_FIELD = "fullName";
+export const DIVULGE_INFO_FIELD = "shareIdentity";
```

Comment:
Could this be renamed to `SHARE_IDENTITY_FIELD` please?

Reply:
Thanks Tori! Fix done